src/library/onlineStatusContext.js

```diff
@@ -17,24 +17,25 @@ const OnlineStatusProvider = ({ children, value }) => {
 
   const rePingApiRef = useRef()
 
-  const stopPingingApi = useCallback(() => {
```

Comment:
I dont know why this is in a callback.... it shoudl live inside the effect. Maybe the code evolved...